net/sslh/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		sslh
+PLUGIN_VERSION=		1.21c
+PLUGIN_REVISION=	1
+PLUGIN_COMMENT=		sslh configuration front-end
+PLUGIN_DEPENDS=		sslh
+PLUGIN_MAINTAINER=	7823088+agh1467@users.noreply.github.com
```

Comment:
I've dropped a mail into this direction. Unwilling to use it if you cannot be reached.

Reply:
nope, message bounced

Reply:
@fichtner This is an "email address" that is used by github to connect commits to a specific user when the user desires not to expose his / her email address in commits. I guess there is no forwarding by github.

Reply:
@agh1467 thanks, highly appreciated